lms/templates/courseware/course_about.html

```diff
@@ -14,9 +14,20 @@
 
 <%inherit file="../main.html" />
 <%block name="headextra">
-  ## OG (Open Graph) title and description added below to give social media info to display
+  <%
+    site_domain = static.get_value('site_domain', settings.SITE_NAME)
+    site_protocol = 'https' if settings.HTTPS == 'on' else 'http'
+
+    og_img_url = "{protocol}://{domain}{path}".format(
+        protocol=site_protocol,
+        domain=site_domain,
+        path=course_image_urls['large']
```

Comment:
Does this handle the case where the course doesn't have an image set?

Reply:
Hi @pdpinch 
If the course doesn't have the image set - then it falls back to the default (copied from common.py lms's settings):
```
DEFAULT_COURSE_ABOUT_IMAGE_URL = 'images/pencils.jpg'
```

You can check the `def image_urls(self)` property from the `CourseOverview` model for details on how the `course_image_urls` dict is populated.

In other words - the handling is the same as for the course image on the course about page.